src/app/pages/upload/DataUpload.js

```diff
@@ -1,48 +1,88 @@
-import React, {Component} from 'react';
-import BrowseImage from "./BrowseImage";
+import React, {Component, createRef} from 'react';
+import {_Post, _PostUpload} from "../../../services/UploadService";
 
 class DataUpload extends Component {
-    state= {
+
+    state = {
         text: '',
-        imageSrc: '',
-        disabled: true,
+        seclectedImage: [],
+        loading: true,
```

Comment:
để phân biện biến kiểu bool với các kiểu khác, thường hay có theo is ở trước
ví dụ thay vì đặt loading thì đặt isLoading. người đọc sẽ hiểu ngay là kiểu boolean có giá trị true hoặc false

Reply:
seclectedImage là mảng thì nên là 1 danh từ số nhiều -> seclectedImages